app/actions/websocket/scheduled_post.test.ts

```diff
@@ -51,48 +35,63 @@ afterEach(async () => {
 });
 
 describe('handleCreateOrUpdateSchedulePost', () => {
```

Comment:
It would be nicer to have the tests check if the posts were really created / deleted in the database, but 0/5

Reply:
Done.